ecosystem/sep-0010.md

```diff
@@ -107,8 +107,8 @@ To validate the challenge transaction the following steps are performed by the s
 * verify that transaction source account is equal to the server's signing key;
 * verify that transaction has time bounds set, and that current time is between the minimum and maximum bounds;
 * verify that transaction contains a single [Manage Data](https://www.stellar.org/developers/guides/concepts/list-of-operations.html#manage-data) operation and it's source account is not null;
-* verify that transaction envelope has a correct signature by server's signing key;
-* verify that transaction envelope has a correct signature by the operation's source account;
+* verify that transaction envelope has signatures from valid signers of the transaction's source account with summed weight greater than the medium threshold on the transaction's source account;
```

Comment:
I see. I read the SEP as saying that the anchor needed to sign with a key for the issuing account. I am ok with the signing account instead though.

Reply:
I think that spec explicitly says about `SIGNING_KEY` usage [here](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0010.md#challenge). It makes sense because this spec targets any web service which needs to authenticate a user via Stellar wallet, not specifically anchors. So the whole concept of "issuing account" is kind of meaningless here.

Reply:
Yeah, I fixed it. Some issuers might also have their issuing accounts locked.

Reply:
Great point!